Mathlib/MeasureTheory/Function/LocallyIntegrable.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Floris van Doorn
 -/
 import Mathlib.MeasureTheory.Integral.IntegrableOn
+import Mathlib.MeasureTheory.Integral.Bochner
```

Comment:
@sgouezel I looked into it, and I couldn't find another file that is appropriate to these lemmas. I could move them to `Bochner.lean` but I think that would crowd it too much with topological imports. What do you think?

Reply:
At least for the first lemma, I think all the topological imports are already there in `Bochner.lean`. So that looks like the right place for it. The second one could for instance go in `SetIntegral.lean`.

Reply:
@sgouezel The second lemma doesn't talk about integrals on a subset, so I don't think `SetIntegral` is appropriate for it.

Reply:
What really matters is to have a file with the right imports, and where it's not crazy to put your lemma in. `SetIntegral` corresponds rather closely to this description (there are several other lemmas in this file which are not talking about integrals on subsets, but whose proof uses lemmas on integrals on subsets, so you don't need a 100% correspondence with the theme of the file).

Reply:
@sgouezel Very well. I've moved the lemmas. Does it look OK now?